lib/msal-node/src/cache/ICachePlugin.ts

```diff
@@ -0,0 +1,6 @@
+import { JsonCache } from '@azure/msal-common';
+
+export interface ICachePlugin {
```

Comment:
This is what the extension library, or any developer wanting to implement some kind of persistence can implement. 

Open to name changes